src/wp-includes/block-supports/layout.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
This one is only half backported, because it's only used for themes with theme.json which is not in Core yet. So for now I only included the mandatory filter to make themes without theme.json work properly.